src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -280,6 +280,51 @@ std::string PolicyManagerImpl::GetLockScreenIconUrl() const {
   return cache_->GetLockScreenIconUrl();
 }
 
+/**
+ * @brief FilterInvalidFunctions filter functions that are not present in schema
+ * @param rpcs list of functins to filter
+ */
+void FilterInvalidFunctions(policy_table::Rpc& rpcs) {
+  policy_table::Rpc valid_rpcs;
+  for (auto& rpc : rpcs) {
```

Comment:
@LuxoftAKutsan `for (const auto& rpc: rpcs)` ?

Reply:
@AByzhynar fixed 829bec8